### ISSUE TITLE
Beta scripts in readme.md

### ISSUE THREAD

**Issue by delsdog** [jamietr1/google-docs-writing-tracker#33]
I presume the scripts in the master branch should be used, I first followed thos referenced in the install scripts section and they are the beta ones (https://raw.githubusercontent.com/jamietr1/google-docs-writing-tracker/beta-version-2/writing_stats.gs) which lack some of the functionality of hte master ones and still contain the deprecated Session.getTimeZone()
Love the scripts though, really appreciate the work that's been put into them.

**Comment by jamietr1** [comment]
Yes, @delsdog, the master branch always contains the current "released" code. Other branches may have newer code, but it is either stuff that I am still working on, or stuff that I haven't completely tested yet.
